### ISSUE TITLE
2.4.11: Text below figure figure 11

### ISSUE THREAD

**Issue by benja11y** [w3c/wcag#1865]
This failing example also fails "Adjacent contrast", which references if you add a border or outline of a very similar color.  We feel that a reference to this should be made in order to avoid authors changing "light-gray" to "dark-gray" to meet this 3:1 requirement without meeting the "Adjacent contrast" requirement.

This issue has been raised on behalf of [TPGi](https://www.tpgi.com), based on an internal consultation process. [An article containing all of our comments](https://www.tpgi.com/wcag-22-working-draft-feedback/) can be found on our blog, along with an introduction and acknowledgments.

**Comment by alastc** [comment]
I don't think it does, as the adjacent background for the button is black, so the light grey does meet "3:1 against adjacent colors in the focused component".

**Comment by DavidMacDonald** [comment]
@alastc 
Actually I think the example does fail 1.4.11 because the light grey focus indicator becomes part of the button in that SC and that makes the adjacent colors of the button(with focus indicator) against the white background 1.6:1 
I think we should either remove the example because its already covered in 1.4.11 or make some clarification

![image](https://user-images.githubusercontent.com/2365411/124309792-44dec100-db39-11eb-91f0-aac0a45e8e6d.png)

![image](https://user-images.githubusercontent.com/2365411/124309991-8f603d80-db39-11eb-8ace-548563698449.png)

**Comment by mraccess77** [comment]
To David’s point I couldn’t even see that it was there.

**Comment by alastc** [comment]
Hi @DavidMacDonald  - the question is about [this figure in the understanding doc](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum.html#focus-indicator-non-contrast) for focus appearance (not non-text contrast).

It is a light grey indicator, adjacent to the white background (which we agree is non-contrasting), and a black border of the component.

The original topic was:

> This failing example also fails "Adjacent contrast", which references if you add a border or outline of a very similar color.

If you compare the indicator to the component, that's #000 with #ccc (13.1), so the adjacent contrast is good against the component.

The adjacent contrast is specifically against the component:
> Adjacent contrast: The contrasting area also has a contrast ratio of least 3:1 against adjacent colors **in the focused component**

I still need to bring back that NTC overlap question, but this question is on focus appearance.

**Comment by GreggVan** [comment]
Two thoughts here.

Using WCAG Conformance model — where you need to meet all of the provisions at a conformance level -  we did not worry about edge cases in one provision if it clearly violated another provision at the same level.    In fact, a number of the provisions were designed with "overlapping fields of coverage" where we ensured that the combination of provisions got all the coverage needed.  

In this case - if a problem is taken care of with one provision -  it would not matter if another provision’s wording would otherwise let something slip by. That appears to be the case here if I understand it.

Clearly — if the provision just says that the focus needs to contrast with the control (and not the background as well) then a focus that is the same color as the background (or close to it) will pass the "contrast with the control" but not "contrast with the background.". 

If covered by another provision - no problem.  
If not - big problem
If another conformance model is used - then big problem.
       — PS if conformance model changes then other parts of WCAG need to be rethought - since the SC were created with the understanding that provisions worked together to cover issues.    They were meant to be taken as a whole - not as individual provisions where only some or most are met. 


gregg

———————————
Professor, University of Maryland, College Park
Director , Trace R&D Center, UMD

**Comment by alastc** [comment]
Closing as I think this is a non-issue (and Ben is a group member). @benja11y if you disagree please re-open, but check my response above first.
